diff -urN aa/jamboree-0.4/configure.in jamboree-0.4/configure.in
--- aa/jamboree-0.4/configure.in	2004-02-07 14:47:47.000000000 +0100
+++ jamboree-0.4/configure.in	2004-03-10 18:48:47.649070304 +0100
@@ -91,7 +91,7 @@
 GST_VER=0.7.4
 
 if test "x$have_xine" = "xno"; then
-    gst_pkgconfig="gstreamer-0.7 >= $GST_VER gstreamer-control-0.7 >= $GST_VER gstreamer-gconf-0.7 >= $GST_VER"
+    gst_pkgconfig="gstreamer-0.8 >= $GST_VER gstreamer-control-0.8 >= $GST_VER gstreamer-gconf-0.8 >= $GST_VER"
 else
     gst_pkgconfig=""
 fi
