diff -urN jamboree-0.5.oryg/configure.in jamboree-0.5/configure.in
--- jamboree-0.5.oryg/configure.in	2004-09-28 13:00:45.031327744 +0200
+++ jamboree-0.5/configure.in	2004-09-28 13:19:55.006504944 +0200
@@ -130,7 +130,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [ Gettext package ])
 
-ALL_LINGUAS="az ca cs de en_CA en_GB es fi fr hr nl no pt pt_BR sr sr@Latn sv"
+ALL_LINGUAS="az ca cs de en_CA en_GB es fi fr hr nb nl pt pt_BR sr sr@Latn sv"
 AM_GLIB_GNU_GETTEXT
 AM_GLIB_DEFINE_LOCALEDIR(JAMBOREE_LOCALEDIR)
 
