diff -urN jamboree-0.5.oryg/configure.in jamboree-0.5/configure.in
--- jamboree-0.5.oryg/configure.in	2004-09-28 13:37:04.225039944 +0200
+++ jamboree-0.5/configure.in	2004-09-28 13:50:21.013909624 +0200
@@ -85,38 +85,7 @@
 sinks=
 AM_GST_ELEMENT_CHECK(esdsink,sinks="${sinks}esdsink ")
 AM_GST_ELEMENT_CHECK(osssink,sinks="${sinks}osssink ")
-
-dnl Check for available decoders
-dnl ============================
-missing_mp3=
-AM_GST_ELEMENT_CHECK(id3demux,,missing_mp3="$missing_mp3 id3demux")
-AM_GST_ELEMENT_CHECK(mad,,missng_mp3="$missing_mp3 mad")
-
-missing_ogg=
-AM_GST_ELEMENT_CHECK(oggdemux,,missing_ogg="$missing_ogg oggdemux")
-AM_GST_ELEMENT_CHECK(vorbisdec,,missing_ogg="$missing_ogg vorbisdec")
-
-formats=
-if test x"$missing_mp3" = x; then
- AC_DEFINE(HAVE_MP3, 1, mp3 support)
- formats="${formats}mp3 "
-fi
-
-
-if test x"$missing_ogg" = x; then
- AC_DEFINE(HAVE_OGG, 1, ogg support)
- formats="${formats}ogg "
-fi
-
-if test x"$formats" = x; then
-  AC_MSG_ERROR([
-
-There are no GStreamer elements installed for mp3 or Vorbis ogg support.
-
-Make sure you have the appropriate packages installed.
-])
-fi
-
+AM_GST_ELEMENT_CHECK(alsasink,sinks="${sinks}alsasink ")
 
 dnl Check for gdbm
 dnl ==========================
